diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/generation/Generation.kt

```diff
@@ -18,6 +20,7 @@ private val autoGenerationComment =
 
 fun main() {
     generateWarningNames()
+    generateCodeStyle()
```

Comment:
I'm starting to think that we need a separate file for such tasks with documentation in order not to run them during every build, maybe in a separate maven profile. But it can be done later in a separate PR.